runtime/stdlib/account.go

```diff
@@ -784,7 +792,12 @@ func newAccountKeysCountConstructor(
 
 	return func() interpreter.UInt64Value {
 		return interpreter.NewUInt64Value(gauge, func() uint64 {
-			return provider.AccountKeysCount(address)
+			count, err := provider.AccountKeysCount(address)
+			if err != nil {
+				// if the FVM isn't able to fetch the number of account keys, we have a problem
```

Comment:
Maybe generalize "FVM" to "provider" – the embedder of Cadence might not be FVM, but e.g. some other blockchain implementation.

```suggestion
				// The provider might not be able to fetch the number of account keys, e.g when the account does not exist.
				// Panic with the user error
```